diff --git a/node_modules/react-native-auth0/A0Auth0.podspec b/node_modules/react-native-auth0/A0Auth0.podspec
index bafcf88..6e0d923 100644
--- a/node_modules/react-native-auth0/A0Auth0.podspec
+++ b/node_modules/react-native-auth0/A0Auth0.podspec
@@ -9,7 +9,7 @@ Pod::Spec.new do |s|
   s.homepage     = package['repository']['baseUrl']
   s.license      = package['license']
   s.authors      = package['author']
-  s.platforms    = { :ios => '13.0' }
+  s.platforms    = { :ios => '13.4' }
   s.source       = { :git => 'https://github.com/auth0/react-native-auth0.git', :tag => "v#{s.version}" }
 
   s.source_files = 'ios/**/*.{h,m,mm,swift}'
